cf-uaac.gemspec

```diff
@@ -24,22 +24,20 @@ Gem::Specification.new do |s|
   s.summary     = %q{Command line interface for CloudFoundry UAA}
   s.description = %q{Client command line tools for interacting with the CloudFoundry User Account and Authorization (UAA) server.  The UAA is an OAuth2 Authorization Server so it can be used by webapps and command line apps to obtain access tokens to act on behalf of users.  The tokens can then be used to access protected resources in a Resource Server.  This library can be used by clients (as a convenient wrapper for mainstream oauth gems) or by resource servers.}
 
-  s.rubyforge_project = 'cf-uaac'
-
   s.license       = 'Apache-2.0'
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ['lib']
 
   # dependencies
-  s.add_runtime_dependency 'cf-uaa-lib', '~> 3.11'
+  s.add_runtime_dependency 'cf-uaa-lib', '~> 3.14', '> 3.14.3'
```

Comment:
rebase to 3.13.4

Reply:
I think we'd better accept any non breaking 3.x version of the library with the `~> 3.14` requirement, and still force avoiding anything non-Ruby 3 compliant with the second `> 3.14.3` requirement.
So, I suggest we now use this new version requirements.